introduction.md

```diff
@@ -1 +1,4 @@
-I'm the new disciple and a JavaScript ninja.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?